core/src/main/java/org/pentaho/di/connections/ConnectionManager.java

```diff
@@ -291,6 +293,22 @@ public <T extends ConnectionDetails> boolean test( T connectionDetails ) throws
     return connectionProvider.test( connectionDetails );
   }
 
+  /**
+   * Run a test operation on the named connection
```

Comment:
The connection being named or not does not seem relevant to mention here.
Also, the doclet is missing the options argument.

```js
/**
  * Tests if a given VFS connection is valid, optionally, with certain testing options.
  *
  * @param connectionDetails The VFS connection.
  * @param options The testing options, or {@code null}.
  * @return {@code true} if the connection is valid; {@code false}, otherwise.
  */
```

Reply:
changes addressed.